### ISSUE TITLE
Day selection does not work for recurring weekly event

### ISSUE THREAD

**Issue by a11ydoer** [w3c/calendar#10]
Thanks for providing one stop place to manage all the W3C meetings! This will be a very handy tool. 

I tested to create ARIA APG recurring event, every Tuesday of the week. I was not able to select the date in addition to keyboard only. I used the most recent chrome , v 89.0.4389.82 in Mac OS 11.2.1 to create the event.

**Comment by jean-gui** [comment]
Hi,

Is that the start date, end date and recurring event's "until" that you cannot change? Those are HTML5 date input fields and I'm able to change them, with either mouse or keyboard only on latest versions of Firefox and Chrome (but on Linux).
Would you happen to have any insight on what could be wrong?

**Comment by a11ydoer** [comment]
@jean-gui  I found that it is my own user error with the excitement for the tool. My apologies. 

I understood "on" field was dropdown list under "every" field because it didnot have any indication of value type for "every".  I also did not consider increase or decrease control for number of week(s).
![UI confusion as dropdown selction list](https://user-images.githubusercontent.com/14183123/111391142-00124580-8682-11eb-91b7-ad92a72372cb.png)

One way to deal with this confusion may be adding more spaces between "every" field and "on" field as well as widen "on: date selection box width. 
![UI suggestion](https://user-images.githubusercontent.com/14183123/111392287-3f419600-8684-11eb-842f-c24b6ff1eebf.png)

Thanks so much again for this exciting tool, @jean-gui

**Comment by jean-gui** [comment]
Thanks for the detailed reply. I'll try to improve the display of this field.

**Comment by jean-gui** [comment]
@a11ydoer I've applied your suggested changes. How does https://www.w3.org/events/meetings/new look now?
